iocore/net/SSLCertLookup.cc

```diff
@@ -130,8 +132,7 @@ struct SSLContextStorage {
 
   /// We can only match one layer with the wildcards
   /// This table stores the wildcarded subdomain
-  InkHashTable *wilddomains;
-
+  mutable HashMap<cchar *, StringHashFns, const int *> wilddomains;
```

Comment:
Maybe I'm confused, but why does this have to be an int* ? Cant the container store a POD type like int directly? That would avoid both he memory allocations and the free no? I.e. could we not do 

```
mutable HashMap<cchar *, StringHashFns, int> wilddomains;
```